src/indexing.jl

```diff
@@ -145,3 +145,13 @@ function index(indexer::Indexer)
     # check if all relevant files are saved
     _check_all_files_are_saved(indexer.config.index_path)
 end
+
```

Comment:
For all `Base.show` methods, since we're only dealing with text, can you add the MIME type in the `show` method as well? For reference you can check [this](https://stackoverflow.com/questions/58962304/how-to-overload-base-show-for-custom-array-types) or [this](https://discourse.julialang.org/t/extending-base-show-for-array-of-types/31289/2).